### PR TITLE
Update en-en.json

### PR DESCRIPTION
**Pull request by meneramandarin**
language changes

### CODE REVIEW

File: intl/locales/en-en.json [hunk 1]
```diff
@@ -382,13 +382,13 @@
   "investment-flow.bank-transfer-summary.neu-reward": "estimated neu reward",
   "investment-flow.bank-transfer.account-name": "account name",
   "investment-flow.bank-transfer.amount": "amount to send",
-  "investment-flow.bank-transfer.bank-details-instructions": "Funds must be transferred from a bank account matching the verified users' name and country of this account and including your reference number. <strong>If the below does not match the sending bank account your funds will be returned and no tokens will be issued.</strong>",
+  "investment-flow.bank-transfer.bank-details-instructions": "Funds must be transferred from a bank account matching the verified users' name and country of this account and include your reference number. <strong>If the information below does not match with the bank account you are wireing funds from your funds will be returned and no tokens will be issued.</strong>",
```
Comment (desfero) [inline intl/locales/en-en.json]: @meneramandarin i'm far from being native in English but `you are **wireing** funds from` sounds incorrect? :)

File: intl/locales/en-en.json [hunk 2]
```diff
@@ -382,13 +382,13 @@
...
   "investment-flow.bank-transfer.gas-stipend": "I agree to increase my investment by a 10 EUR service fee. The fee will be converted into ETH to pay for future blockchain transactions on the platform.",
-  "investment-flow.bank-transfer.general-instructions": "Investments via bank transfer take approx. 2 to 3 days. Once received we will send you an email confirmation and will issue your tokens to your account. If the investment is not received before the end of the sale your money will be returned to your bank account within 1 to 2 working days.",
+  "investment-flow.bank-transfer.general-instructions": "Investments via bank transfer take approx. 4 to 7 business days. In order to complete the transfer, please include the reference number on your. Without this we might not be able to connect the transferred funds with your account. Your funds will be converted to EUR and automatically invested into the Fifth Force ETO. Once received we will send you an email confirmation and will issue the tokens to your account. If the investment is not received before the end of the sale your money will be returned to your bank account within 1 to 2 working days.",
```
Comment (desfero) [inline intl/locales/en-en.json]: ` please include the reference number on your` seems like a not a finished sentence (at least i can't catch where should i include reference number)

File: intl/locales/en-en.json (hunk 1 above)
Comment (meneramandarin) [reply intl/locales/en-en.json]: you're right it's wiring

File: intl/locales/en-en.json (hunk 2 above)
Comment (meneramandarin) [reply intl/locales/en-en.json]: please include the reference number when making the bank transfer. without the on your

File: intl/locales/en-en.json (hunk 1 above)
Comment (desfero) [reply intl/locales/en-en.json]: fixed

**Pull request merged**